spec/rollbar/middleware/sinatra_spec.rb

```diff
@@ -42,7 +42,7 @@ def app
   end
 
   let(:expected_report_args) do
-    [uncaught_level, exception]
+    [uncaught_level, exception, { :use_exception_level_filters => true}]
```

Comment:
not, it should be `{ :use_exception_level_filters => true }` as most of ruby style guides recommend.

I'll fix it.